django/chat/views.py

```diff
@@ -542,7 +541,7 @@ def chat_options(request, chat_id, action=None, preset_id=None):
         # Update the chat options with the preset options
         copy_options(preset.options, chat.options)
 
-        chat_options_form = ChatOptionsForm(instance=preset.options, user=request.user)
+        chat_options_form = ChatOptionsForm(instance=chat.options, user=request.user)
```

Comment:
Because copy_options may have reset the QA library, its now important to pass the updated chat.options instead of the source preset.options